web/client/utils/PluginsUtils.js

```diff
@@ -122,6 +124,11 @@ const getReducers = (plugins) => Object.keys(plugins).map((name) => plugins[name
 const getEpics = (plugins) => Object.keys(plugins).map((name) => plugins[name].epics)
                             .reduce((previous, current) => assign({}, previous, current), {});
 
+const pluginsMergeProps = (stateProps, dispatchProps, ownProps) => {
+    const {pluginCfg, ...otherProps} = ownProps;
+    return assign({}, otherProps, stateProps, dispatchProps, pluginCfg || {});
```

Comment:
Didn't understand how to configure it.
pluginsCfg always wind over stateProps?
Could it be that pluginsCfg can be overridden by stateProps? Or do you mean that all the other Props are overridable while pluginCfg is the way to set fixed config. 

can you document it?